dd-sdk-android/src/test/kotlin/com/datadog/android/log/assertj/LogAssert.kt

```diff
@@ -117,10 +117,6 @@ internal class LogAssert(actual: Log) :
 
     fun hasUserInfo(expected: UserInfo?): LogAssert {
         assertThat(actual.userInfo)
-            .overridingErrorMessage(
```

Comment:
Is because the forged attributes inside the UserInfo object may contain `%` characters which when that message is formatted internally are making the code to crash because it is expecting an argument for that `%` character. I could sanitise the error message but I'd really prefer not to do it and let the `AssertJ` library use it's default error message in this case.